perfzero/lib/benchmark.py

```diff
@@ -69,6 +69,12 @@ def _setup(self):
     self.benchmark_execution_time['checkout_repository'] = (
         time.time() - start_time)
 
+    # Start cloud TPU.
+    if self.config.tpu_parameters is not None:
+      start_time = time.time()
+      utils.setup_tpu(self.config.tpu_parameters)
```

Comment:
setup_tpu() internally should have:

def setup_tpu():
   try:
     ..run setup()
   except Exception:
     run_cleanup()
    sys.exit(1)

Reply:
Thanks, added this try-except logic in setup_tpu now.

The cleanup step is currently just "rm -f ctpu" in case a bad ctpu tool has been downloaded and causes errors. Based on past tests with "ctpu up", if the command fails (e.g., due to bad params, insufficient quota), the TPU is not created and we don't need to delete the TPU.